test/extended/fixtures/forcepull-false-cust.json

```diff
@@ -0,0 +1,384 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "ruby-helloworld-sample",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "This example shows how to create a simple ruby application in openshift origin v3"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
```

Comment:
Yeah, I kinda went for the full bore on purpose, thinking the extra moving parts had a better chance of turning up obscure bugs.  Or at least let's have a mixture.  Some of the tests leverage the stripped down approach, others perhaps don't.

Thoughts?